extensions/json/internal.m

```diff
@@ -105,7 +104,7 @@ - (id)decodeFromFile:(NSString *)path {
         return nil;
     }
 
-    return [self decode:json];
+    return [self decode:[json dataUsingEncoding:NSUTF8StringEncoding]];
```

Comment:
Maybe here instead of doing `NSString stringWithContentsOfFile` we should just do `NSData dataWithContentsOfFile`?